ocw/utils.py

```diff
@@ -335,5 +337,11 @@ def calc_climatology_monthly(dataset):
         )
         raise ValueError(error)
     else:
-        return reshape_monthly_to_annually(dataset).mean(axis=0)
+        values = reshape_monthly_to_annually(dataset).mean(axis=0)
+        
+        # A year can commence from any month
+        first_month = dataset.times[0].month
```

Comment:
Hey Kim,

Just to make sure I understand what's going on here. You're trying to end up with an array that contains all the months for a particular year right? This is going to carry over into a following year. Is that going to be problematic or do we just not care?

Also, being totally nitpicky but ```relativedelta(months=x)``` is better since it's a keyword arg and we don't put spaces around those. Easy enough to fix later though. =D